server/src/test/java/org/apache/druid/server/http/CoordinatorDynamicConfigTest.java

```diff
@@ -163,15 +173,16 @@ public void testDeserializeWithoutMaxSegmentsInNodeLoadingQueue() throws Excepti
         CoordinatorDynamicConfig.class
     );
 
-    assertConfig(actual, 1, 1, 1, 1, 1, 1, 2, true, ImmutableSet.of(), true, 0);
+    assertConfig(actual, 1, 1, 1, 1, 1, 1, 2, true, ImmutableSet.of(), true, 0, ImmutableSet.of(), 0);
   }
 
   @Test
   public void testBuilderDefaults()
   {
 
     CoordinatorDynamicConfig defaultConfig = CoordinatorDynamicConfig.builder().build();
-    assertConfig(defaultConfig, 900000, 524288000, 100, 5, 15, 10, 1, false, ImmutableSet.of(), false, 0);
+    assertConfig(defaultConfig, 900000, 524288000, 100, 5, 15, 10, 1, false, ImmutableSet.of(), false, 0, ImmutableSet.of(),
+                 7);
```

Comment:
According to the Druid formatting rules, should break all or none arguments.

Reply:
:white_check_mark: